### ISSUE TITLE
does gunicorn supports python 3.10 environment?

### ISSUE THREAD

**Issue by vikramcse** [benoitc/gunicorn#3019]
Hello

does gunicorn supports python 3.10 environment? as per the pypy gunicorn, it support till python 3.9
![image](https://github.com/benoitc/gunicorn/assets/1611331/9aa3e93f-3f6b-490c-bcb3-0d5f384c79d7)

**Comment by benoitc** [comment]
Yes it is: https://github.com/benoitc/gunicorn/blob/master/setup.py#L29-L30

new release is out this friday.

**Comment by vikramcse** [comment]
Hello @benoitc 
Thanks for your response
I checked in PyPI and I did not found the the release for gunicorn 3.10

**Comment by benoitc** [comment]
i'm aware the release is not been done yet. 

master is available waiting that as i am side tracked by **customers** today. A public  release will happen this week in any case.

**Comment by benoitc** [comment]
this has been released
